docs/examples/general/tensor_join.ipynb

```diff
@@ -4,22 +4,26 @@
    "cell_type": "markdown",
```

Comment:
<p>Bullet point list is malformed in html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>